--- a/src/htmlparser/fsm_config.py	2014-03-19 15:34:51.000000000 -0400
+++ b/src/htmlparser/fsm_config.py	2023-04-17 14:07:56.000000000 -0400
@@ -62,13 +62,13 @@

   # Below are all we have to define in addition to what DictMixin would need
   def __len__(self):
-    return len(self.keys())
+    return len(list(self.keys()))

   def __contains__(self, key):
-    return self.has_key(key)
+    return key in self

   def __iter__(self):
     # It's not as portable -- though it would be more space-efficient -- to do
     #   for k in self.keys(): yield k
-    return iter(self.keys())
+    return iter(list(self.keys()))

